make style returns 5 errors

Hi,
My experience with `event-sourcing`, `CQRS` and `DDD`are pretty new so please be nice if my questions are stupid.

I take time to dig into your boilerplate and I run into the command `make style`. It comes up with those errors :

```
docker-compose exec php sh -lc './vendor/bin/phpstan analyse -l 5 -c phpstan.neon src tests'
 85/85 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

 ------ ------------------------------------------------------------------------- 
  Line   src/Infrastructure/User/Auth/Guard/LoginAuthenticator.php                
 ------ ------------------------------------------------------------------------- 
  163    Property App\Infrastructure\User\Auth\Guard\LoginAuthenticator::$router  
         (Symfony\Bundle\FrameworkBundle\Routing\Router) does not accept          
         Symfony\Component\Routing\Generator\UrlGeneratorInterface.               
 ------ ------------------------------------------------------------------------- 

 ------ --------------------------------------------------------------- 
  Line   src/Infrastructure/User/Query/UserReadProjectionFactory.php    
 ------ --------------------------------------------------------------- 
  26     Property App\Domain\User\ValueObject\Auth\Credentials::$email  
         (App\Domain\User\ValueObject\Email) does not accept string.    
 ------ --------------------------------------------------------------- 

 ------ ----------------------------------------------------------------- 
  Line   src/Infrastructure/Share/Query/Repository/ElasticRepository.php  
 ------ ----------------------------------------------------------------- 
  58     Array (array<string>) does not accept int.                       
 ------ ----------------------------------------------------------------- 

 ------ ------------------------------------------------------------ 
  Line   tests/UI/Http/Web/Controller/ProfileControllerTest.php      
 ------ ------------------------------------------------------------ 
  27     Call to an undefined method                                 
         Symfony\Component\HttpFoundation\Response::getTargetUrl().  
 ------ ------------------------------------------------------------ 

 ------ ------------------------------------------------------------------ 
  Line   tests/UI/Http/Rest/Controller/Events/GetEventsControllerTest.php  
 ------ ------------------------------------------------------------------ 
  37     Casting to string something that's already string.                
 ------ ------------------------------------------------------------------ 

                                                                                
 [ERROR] Found 5 errors                                                         
                                                                                

make: *** [style] Error 1
```

Are configuration errors on my laptotp ? Something else ?

For now I did'nt do anything, just installed your boilerplate

--- a/makefile
+++ b/makefile
@@ -28,7 +28,7 @@ phpunit: db ## execute project unit tests
 
 .PHONY: style
 style: ## executes php analizers
-		docker-compose exec php sh -lc './vendor/bin/phpstan analyse -l 5 -c phpstan.neon src tests'
+		docker-compose exec php sh -lc './vendor/bin/phpstan analyse -l 6 -c phpstan.neon src tests'
 
 .PHONY: db
 db: ## recreate database

--- a/src/Infrastructure/Share/Query/Repository/ElasticRepository.php
+++ b/src/Infrastructure/Share/Query/Repository/ElasticRepository.php
@@ -54,6 +54,8 @@ abstract class ElasticRepository
     {
         Assertion::greaterThan($page, 0, 'Pagination need to be > 0');
 
+        $query = [];
+
         $query['index'] = $query['type']  = $this->index;
         $query['from']  = ($page - 1) * $limit;
         $query['size']  = $limit;

--- a/src/Infrastructure/User/Auth/Guard/LoginAuthenticator.php
+++ b/src/Infrastructure/User/Auth/Guard/LoginAuthenticator.php
@@ -11,7 +11,6 @@ use App\Domain\User\Exception\InvalidCredentialsException;
 use App\Domain\User\Query\UserView;
 use App\Infrastructure\User\Auth\Auth;
 use League\Tactician\CommandBus;
-use Symfony\Bundle\FrameworkBundle\Routing\Router;
 use Symfony\Component\HttpFoundation\RedirectResponse;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
@@ -175,7 +174,7 @@ final class LoginAuthenticator extends AbstractFormLoginAuthenticator
     private $queryBus;
 
     /**
-     * @var Router
+     * @var UrlGeneratorInterface
      */
     private $router;
 }

--- a/src/Infrastructure/User/Query/UserReadProjectionFactory.php
+++ b/src/Infrastructure/User/Query/UserReadProjectionFactory.php
@@ -23,7 +23,7 @@ class UserReadProjectionFactory extends Projector
     {
         $userReadModel = $this->repository->oneByUuid($emailChanged->uuid);
 
-        $userReadModel->credentials->email = $emailChanged->email->toString();
+        $userReadModel->credentials->email = $emailChanged->email;
 
         $this->repository->apply();
     }

--- a/tests/UI/Http/Rest/Controller/Events/GetEventsControllerTest.php
+++ b/tests/UI/Http/Rest/Controller/Events/GetEventsControllerTest.php
@@ -18,7 +18,7 @@ class GetEventsControllerTest extends JsonApiTestCase
      *
      * @group e2e
      */
-    public function events_should_be_present_in_elastic_seach()
+    public function events_should_be_present_in_elastic_search()
     {
         $this->post('/api/users', [
             'uuid' => $uuid = Uuid::uuid4()->toString(),
@@ -34,7 +34,7 @@ class GetEventsControllerTest extends JsonApiTestCase
 
         self::assertEquals(200, $this->client->getResponse()->getStatusCode());
 
-        self::assertContains('UserWasCreated', (string) $this->client->getResponse()->getContent());
+        self::assertContains('UserWasCreated', $this->client->getResponse()->getContent());
     }
 
     private function refreshIndex()

--- a/tests/UI/Http/Web/Controller/ProfileControllerTest.php
+++ b/tests/UI/Http/Web/Controller/ProfileControllerTest.php
@@ -6,7 +6,6 @@ namespace App\Tests\UI\Http\Web\Controller;
 
 use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
 use Symfony\Component\HttpFoundation\RedirectResponse;
-use Symfony\Component\HttpFoundation\Response;
 
 class ProfileControllerTest extends WebTestCase
 {
@@ -21,7 +20,7 @@ class ProfileControllerTest extends WebTestCase
 
         $client->request('GET', '/profile');
 
-        /** @var Response|RedirectResponse $response */
+        /** @var RedirectResponse $response */
         $response = $client->getResponse();
         $this->assertEquals(302, $response->getStatusCode());
         $this->assertContains('/sign-in', $response->getTargetUrl());

